### PR TITLE
Added files

### PR DESCRIPTION
**Pull request by Torgian**
Files for all the modified design requirements.

### CODE REVIEW

**Review by martinheidegger — CHANGES_REQUESTED** [review]
This PR contains the content of the entire ghost site. Can you limit it to theme folder?

**Comment by Torgian** [comment]
> This PR contains the content of the entire ghost site. Can you limit it to theme folder?

Sure, though you will have to add some logos through the administration page, and change the description as well.

**Comment by Torgian** [comment]
Ok done. It only has the theme folder now.

**Review by martinheidegger — CHANGES_REQUESTED** [review]
> Sure, though you will have to add some logos through the administration page, and change the description as well.

The ghost setup is based on just referencing the theme from this git repository. Can you list the things that need to be changed on the ghost instance in order for it to look like you intend to? _(Attach Image ; Note where to change what text)_

> Ok done. It only has the theme folder now.

Thank you for putting the only the website  please put the website in the repository root, instead of a "consento-website-master" folder?

Also, I noticed that you _just_ added a new commit with all files. This technically keeps a copy of all the files in the git history and slows downs any pulls from the server. When we merge the PR it can be squashed into one commit, but then the commit history and graph looks weird and anyone that fetches all branches would still be downloading the whole ghost instance. It would be good to squash the commits into one and force-push it to your branch.

**Comment by Torgian** [comment]
Understood. Would it be best if I close this pull request and make a new PR with the changes needed?

**Comment by martinheidegger** [comment]
A force-push is okay. _(no need to create a new pr)_

**Comment by Torgian** [comment]
I did a force push.

File: default.hbs [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
     {{!-- Styles'n'Scripts --}}
     <link rel="stylesheet" type="text/css" href="{{asset "main/css/main.css"}}" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Comment (martinheidegger) [inline default.hbs]: ALL files (.js/.css & images) need to be added to the template in order to create a packaged version of the site that runs on Dat.

File: default.hbs [hunk 2]
```diff
@@ -52,10 +54,13 @@
     <script src="{{asset "main/js/breakpoints.min.js"}}"></script>
     <script src="{{asset "main/js/util.js"}}"></script>
     <script src="{{asset "main/js/main.js"}}"></script>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
```
Comment (martinheidegger) [inline default.hbs]: These urls too.

File: home.hbs [hunk 3]
```diff
@@ -0,0 +1,201 @@
+{{!< default}}
+{{#is "home"}}{{/is}}
+{{!-- <header class="tag-header">
+    <h1>Blog</h1>
+    <p>News &amp; thoughts about the consento process.</p>
+</header> --}}
+
+{{!-- <video id="main-video" preload autoplay muted playsinline>
```
Comment (martinheidegger) [inline home.hbs]: These sections can be removed.

File: partials/footer.hbs [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<footer class="page-footer font-small blue pt-4">
+  <div class="container-fluid text-center text-md-left">
+    <div class="row">
+      <div class="col-md-4 mt-md-0 mt-3">
+        <h2 class="">Contact us</h2>
+        <ul class="list-unstyled">
+          <li>
+            <a href="mailto:keepsafe@consento.org">Investors</a>
```
Comment (martinheidegger) [inline partials/footer.hbs]: @dkastl do you think these should be different mail addresses?
@Torgian Maybe add a ?subject=?

File: partials/footer.hbs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          <li>
+            <a href="#!">White Paper upcoming</a>
+          </li>
+        </ul>
+      </div>
+      <div class="col-md-4 mb-md-0 mb-3">
+        <h2 class="">About</h2>
+        <ul class="list-unstyled">
+          <li>
+            <a href="#!">Team</a>
```
Comment (martinheidegger) [inline partials/footer.hbs]: We do have a team and an about page!

File: partials/footer.hbs [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        <h2 class="">Develop</h2>
+        <ul class="list-unstyled">
+          <li>
+            <a href="https://github.com/consento-org/">Git</a>
+          </li>
+          <li>
+            <a href="https://keybase.io/team/consento">Keybase</a>
+          </li>
+          <li>
+            <a href="#!">White Paper upcoming</a>
```
Comment (martinheidegger) [inline partials/footer.hbs]: Add a note that people interested should ask for it in the keybase chat.

**Review by martinheidegger — CHANGES_REQUESTED** [review]
I noticed that all images were added as-is. Please optimize the images. Maybe with ImageAlpha/ImageOptim - they are far too big (in a quick test i was able to safe 32k just by optimizing the consento log) 

It would be nice if the libraries would not just be included in the theme (hard-link) but also combined into single files (script maybe?) but that is candy. Other notes in the review.

**Comment by Torgian** [comment]
In the admin page, in general settings:

Go to title/description and change the description to Data privacy by humans, for humans.

Add the publication icon and publication logo. This is the consento logo (just the Consento logo )

consento-large

**Comment by Torgian** [comment]
I am going to do normal "pushes" until the rest of the designs are done; then I will squash and force push.

**Comment by martinheidegger** [comment]
It is fine to use normal commits. Usually squashes are not important to me, in this case I requested it but I think we will not need them often in future.

**Comment by martinheidegger** [comment]
Currently the header is a bit massive in the browser.

Screen Shot 2020-01-22 at 11 33 09

**Comment by Torgian** [comment]
Yes, I also noticed a couple other small problems while testing on my phone. I’m in the dentist about to get a new filling, so I’ll work on this tonight. I have other appointments this afternoon to take care of.

**Review by martinheidegger — APPROVED** [review]
latestversion, merging into next for now.

**Pull request merged**